packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -104,10 +90,11 @@
 +      effect: NoExecute
 +
 +global:
++  cattle:
++    systemDefaultRegistry: ""
++  dockerRootDirectory: ""
```

Comment:
Would be nice to add a short comment above this one to describe when to use it, ie when you have moved it.

Reply:
Will add a comment. Good idea